repos.d/managarm.yaml

```diff
@@ -0,0 +1,28 @@
+###########################################################################
+# Managarm
+###########################################################################
+
+- name: managarm_x86_64
```

Comment:
Drop x86_64 part, please. For void it's a legacy artifact, in fact all arches should reside in the single repository.